### ISSUE TITLE
Create a logo for CYBLOG

### ISSUE THREAD

**Issue by chaitanyagiri** [Cybros-LNMIIT/CYBLOG#2]
**I'm submitting a ...**
- [ ] bug report
- [x] feature request

put an 'x' inside [ ] to check.

Every great thing has an identity so we also want CYBLOG to be identified by a certain logo.

**Comment by stefanobg** [comment]
I can work on that!

Is there anything you were thinking about, maybe some inspiration?

**Comment by chaitanyagiri** [comment]
@stefanobg Nothing particular, We are open to suggestions.

**Comment by stefanobg** [comment]
Awesome! 
I'll be working on.

**Comment by uivlis** [comment]
Now I'm not sure if the bounty is still open, but here's my work:

![frame](https://user-images.githubusercontent.com/7414543/52467819-6bc68200-2b8f-11e9-8f0b-1ea1e05a660d.png)

**Comment by chaitanyagiri** [comment]
@uivils thank-you for the contribution. I like the logo.

**Comment by uivlis** [comment]
Right, so... Did I win any bounty? Or, maybe you're still accepting contributions?

**Comment by chaitanyagiri** [comment]
I am really sorry but we removed the bounty and its details from the repository but unfortunately I forgot to remove the tag.

**Comment by chaitanyagiri** [comment]
We are accepting contributions though.
